user-org-actor/src/main/java/org/sunbird/organization/dao/OrgEsDaoImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.sunbird.organization.dao;
+
+import java.util.Map;
+import org.sunbird.common.factory.EsClientFactory;
+import org.sunbird.dto.SearchDTO;
+import org.sunbird.es.service.ElasticSearchService;
+import org.sunbird.helper.ElasticSearchHelper;
+import org.sunbird.helper.EsIndex;
+import org.sunbird.response.Response;
+import org.sunbird.util.jsonkey.JsonKey;
+import scala.concurrent.Future;
+
+public class OrgEsDaoImpl implements IOrgDao {
+
+  private ElasticSearchService es =
+      EsClientFactory.getInstance(EsClientFactory.EsClient.REST.getName());
+  private static OrgEsDaoImpl INSTANCE = new OrgEsDaoImpl();
```

Comment:
INSTANCE must not be a member if its not a capitalised string.

Reply:
done